### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: pir_frontend/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (sdonk) [inline pir_frontend/admin.py]: delete this file if not needed

File: pir_frontend/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (sdonk) [inline pir_frontend/tests.py]: please add a test for the view below

File: pir_frontend/views.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import logging
+
+from requests.exceptions import HTTPError
+from django.shortcuts import render
+from pir_frontend.forms import PIRForm
+
+logger = logging.getLogger(__name__)
+
+# Create your views here.
+def form_base(request):
```
Comment (sdonk) [inline pir_frontend/views.py]: This view can be greatly simplified using CBV

File: config/settings.py [hunk 4]
```diff
@@ -25,7 +25,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = [
```
Comment (eadpearce) [inline config/settings.py]: in other repos we have this
```
# As the app is running behind a host-based router supplied by Heroku or other
# PaaS, we can open ALLOWED_HOSTS
ALLOWED_HOSTS = ['*']
```

File: pir_frontend/views.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    context = {}
+
+    if request.method == 'POST':
+        form = PIRForm(request.POST)
+        if form.is_valid():
+            data = form.cleaned_data
+
+            try:
+                form.client.create_report(data)
+            except Exception as e:
```
Comment (sdonk) [inline pir_frontend/views.py]: Catching all is not really good practice. The client should return a specific exception. We need to log these exceptions on Sentry

**Comment by sdonk** [comment]
- Circle Ci and code coverage support need to be added to this repo
- Requirements should be a `.in` file and be compiled by `pip-tools`
- Remove the `SECRET_KEY` from the repo. It should be in an envar
- `DEBUG` to be set from an env var as well and default to `false`

**Pull request closed without merging**